src/robusta/core/sinks/robusta/dal/model_conversion.py

```diff
@@ -82,6 +82,8 @@ def to_evidence_json(
                 structured_data.append({"type": "header", "data": block.text})
             elif isinstance(block, ListBlock):
                 structured_data.append({"type": "list", "data": block.items})
+            elif isinstance(block, JsonBlock):
```

Comment:
Why weren't we sending Json blocks to the UI?
Maybe it makes more sense to add a variable in base block to not send an enrichment to the UI instead of duplicating the block types?